python/tests/arborist/test_urls.py

```diff
@@ -10,45 +10,92 @@
     # python2
     import mock
 
-def test_healthy_call(arborist_client):
-    with mock.patch("gen3authz.client.arborist.client.requests.get") as mock_get:
-        arborist_client.healthy()
-        assert mock_get.called_with(arborist_client._base_url + "/health")
 
+def test_get_resource_call(arborist_client, mock_arborist_request):
+    mock_get = mock_arborist_request({"/resource/a/b/c": {"GET": (200, {"is": 789})}})
+    assert arborist_client.get_resource("/a/b/c") == {"is": 789}
```

Comment:
just confirming that you check the json response to verify the type returned by the client interface (json not ArboristResponse), as opposed to the contents (which are mocked)?

Reply:
(Yes)